movie-blog/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
```

Comment:
This particular test will fail, but since unit testing is of critical importance for building robust and CI/CD-friendly app, I may recommend to refactor this test to check for existence of your actual component as an additional exercise.